src/styles/cards.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  max-height: 408px;
+  transition: transform 300ms;
+
+  font-family: Roboto, sans-serif;
+
+  &__imac {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin: 0 0 4px;
+  }
+
+  &__title:hover {
+    color: #34568b;
+    cursor: pointer;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__quality {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__cost {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__sum {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    text-transform: uppercase;
+    padding: 12px 70px;
+    border-radius: 5px;
+    background: #00acdc;
+    border: 1px solid #00acdc;
+    color: #fff;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 100%;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    &:hover {
+      color: #00acdc;
+      background: #fff;
+      border: 1px solid #00acdc;
+    }
+  }
+}
+
+.card:hover {
+  cursor: pointer;
+  transform: scale(1.2);
```

Comment:
The hover effect scales the card to 1.2 times its original size. This could cause layout shifts and affect the user experience. Consider using a more subtle scale or adding a transition to smooth the effect.